### ISSUE TITLE
Deplaced labels placement are not saved in project

### ISSUE THREAD

**Issue by rCarto** [riatelab/magrit#86]
Create a label layer
Move some labels
Save project
Open project => labels are back to centroids

**Comment by mthh** [comment]
Yep, I tested on previous versions and it was already the case (although reading the code it looks like there is an intention to keep their positions... this will need to be investigated!)

**Comment by mthh** [comment]
Indeed, was a quick fix, most of the logic was already here. I'm releasing 0.9.1 just now and will update the version on our server tonight.
